lib/ci/ci_type_parser.js

```diff
@@ -25,8 +25,8 @@ const CI_TYPE_ENUM = {
 };
 
 const CI_PROVIDERS = {
-  JENKINS: 'jenkins',
```

Comment:
what motivated the name change? i'm not sure it'd be clear to me what "nodejs" CI meant 🤔

Reply:
It doesn't test only `jenkins` anymore, so a new type made more sense. And since the only repo we use ncu and Jenkins is `nodejs/node`, I thought a provider specific for that repo made sense. Happy to hear other suggestions for names though.

An alternative I thought was accepting a list of providers, so for `nodejs/node` we would have `['jenkins', 'actions']`. But that increases complexity.

Although now I realized that maybe libuv also uses ncu? They definitely generate metadata and have a similar landing process.

Reply:
Also FWIW, we stopped using ncu on llnode, which is what motivated me to have more than one provider here. We can probably get rid of this.